build.gradle

```diff
@@ -941,13 +942,15 @@ project(':server') {
 
     implementation libs.slf4jApi
 
-    compileOnly libs.reload4j
+    compileOnly libs.log4j2Api
+    compileOnly libs.log4j2Core
+    compileOnly libs.log4j1Bridge2Api
```

Comment:
Some APIs rely on it, such as `PropertyConfigurator`. However, since we are removing it, we might not need the Log4j bridge anymore. I will test this locally.

Reply:
@chia7712 @frankvicky When I worked on this issue last, it was required to support the log4j 1.x configuration file.